src/site/content/en/blog/fast-ads-matter/index.md

```diff
@@ -0,0 +1,196 @@
+---
+title: Fast ads matter
+subhead: |
+    Learn how improving ad speed can increase revenue and makes users happy.
```

Comment:
FYI @mihajlija this `subhead` misses the second part of the post, which is about how to approach ad speed improvements.